subprojects/gradle/lint-report/build.gradle.kts

```
@@ -41,7 +41,7 @@ gradlePlugin {
 }
 
 tasks.named<Test>("integrationTest").configure {
-    applyOptionalSystemProperty("avito.slack.test.channel")
+    applyOptionalSystemProperty("avito.slack.test.channelid")
```

```suggestion
    applyOptionalSystemProperty("avito.slack.test.channelId")
```
Can we use camelCase? It's easier for reading

---

yes, will fix